src/main/java/org/minions/devfund/marylin/WhackAMole.java

```diff
@@ -0,0 +1,174 @@
+                strBuilderRow.append("  ");
+            }
+            LOGGER.info(strBuilderRow.toString());
+        }
+    }
+
+    /**
+     * Main method to play the game.
+     *
+     * @param args Default Argument.
+     */
+    public static void main(final String[] args) {
+        int numberOfMoles = 10;
+        int numAttemps = 50;
+        int gridLenght = 10;
+
+        //create a 10 by 10 grid
+        WhackAMole whackAMole = new WhackAMole(numAttemps, gridLenght);
+
+        //Place randomly a total of 10 moles
+        for (int i = 0; i < numberOfMoles; i++) {
+            Random rand = new Random();
+            int x = rand.nextInt(numberOfMoles);
+            int y = rand.nextInt(numberOfMoles);
+            StringBuilder strBuilderMessage = new StringBuilder();
+            if (!whackAMole.place(x, y)) {
+                strBuilderMessage.append("There is already a mole at ");
+                strBuilderMessage.append(x);
+                strBuilderMessage.append(",");
+                strBuilderMessage.append(y);
+                strBuilderMessage.append(". Trying again!");
+                i--;
+            } else {
+                strBuilderMessage.append("Mole places at ");
+                strBuilderMessage.append(x);
+                strBuilderMessage.append(",");
+                strBuilderMessage.append(y);
+            }
+            LOGGER.info(strBuilderMessage.toString());
+        }
+
+        //Try to whack 50 times the moles given the x and y
+        //Finish the game if we enter xAxis -1 and yAxis -1 printing the grid
+        //The game ends if the user is able to uncover all the moles or if the user runs out of attempts.
+
+        LOGGER.info("-------------------------------------");
+        whackAMole.printGrid();
+        LOGGER.info("-----You have a maximum of " + numAttemps + " attempts to get all the moles wacked-----");
+
+        int xAxis;
+        int yAxis;
+        Scanner xScanner;
+        Scanner yScanner;
+        for (int i = numAttemps; i > 0; i--) {
+            xScanner = new Scanner(System.in);
+            LOGGER.info("Enter the x coordinates: ");
+            xAxis = xScanner.nextInt();
+
+            yScanner = new Scanner(System.in);
+            LOGGER.info("Enter the y coordinates: ");
+            yAxis = yScanner.nextInt();
+
+            if (xAxis == -1 || yAxis == -1) {
+                LOGGER.info("Giving up!!");
+                whackAMole.printGrid();
+                break;
+            } else {
+                LOGGER.info("State of Game");
+                whackAMole.whack(xAxis, yAxis);
+                whackAMole.printGridToUser();
+            }
+
+            if (whackAMole.molesLeft == 0) {
+                LOGGER.info("All moles were whacked with score: " + whackAMole.score);
+                break;
+            }
+
+            if (whackAMole.attemptsLeft == 0) {
+                LOGGER.info("There are no more attemps with score: " + whackAMole.score);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)